internal/config/config.go

```diff
@@ -80,6 +82,8 @@ func init() {
 		viper.Set("database.password", cfg.Database.Password)
 		viper.Set("database.name", cfg.Database.Name)
 		viper.Set("prometheus.port", cfg.MetricsPort)
+	} else if IsTestRun() {
+		viper.Set("featureFlags.environment", "test")
```

Comment:
Isn't this missing a new field in the config type? Then add it to the example YAML config. Or I am not following.

Reply:
This has already a key in the yaml file, we just only had developement and production, no test.